backend/estimate-service/pom.xml

```diff
@@ -0,0 +1,120 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Has this passed the snyk vulnerability check?

Reply:
yes.

Reply:
Doesn't this version of spring have the log4j vulnerability issue?

Reply:
no. But its better to add this "<log4j2.version>2.17.1</log4j2.version>" dependency. Shall i go ahead?

Reply:
Yes. Add it